src/main/java/com/mraof/minestuck/world/lands/terrain/RockLandType.java

```diff
@@ -82,27 +89,30 @@ public void setProperties(LandProperties properties)
 	@Override
 	public void setGenSettings(LandGenSettings settings)
 	{
-		settings.oceanChance = 1/4F;
+		settings.oceanChance = 1 / 4F;
 	}
 	
 	@Override
 	public void setBiomeSettings(LandWrapperBiome biome, StructureBlockRegistry blocks)
 	{
+		int biomeMultiplier = 1;
```

Comment:
I'd suggest defining this value directly on this line instead of indirectly with the ++ calls.
`int biomeMultiplier = biome.type == BiomeType.NORMAL ? 1 : 2;`
or
`int biomeMultiplier = biome.type == BiomeType.ROUGH || biome.type == BiomeType.OCEAN ? 2 : 1;`

Reply:
Changed to the second iteration you made there and then also modified the ratio a bit